feat(changelog): jobs-fixed-serverless-jobs-fix-usage-of-cached-images 2024-03-12 (#2868)
scaleway/docs-content: changelog/march2024/2024-03-12-jobs-fixed-serverless-jobs-fix-usage-of-cached-images.mdx

---
title: Serverless Jobs fix usage of cached images
status: fixed
author:
    fullname: 'Join the #serverless-jobs-beta channel on Slack.'
    url: 'https://slack.scaleway.com'
date: 2024-03-12
category: serverless
product: jobs
---

> When running a Serverless Job, it will now run the correct image in all circumstances instead of cached image.

Reviewer: ```suggestion
When running a Serverless Job, the correct image will now be used in all circumstances instead of the cached image.
```
